Automattic/jetpack
# Adding BottomSheet settings for Tiled Gallery

Work for 
- https://github.com/wordpress-mobile/gutenberg-mobile/issues/3947

#### Changes proposed in this Pull Request:

* Adds settings (Rounded corners, columns, link to) to Tiled Gallery

#### Does this pull request change what data or activity we track or use?
My PR does not change any tracked data/activity.

#### Screenshots
https://user-images.githubusercontent.com/13263478/132838350-1b463ccc-ae57-4f9d-8175-2df6c81812eb.mp4

#### Testing instructions:
* Add tiled gallery block
* Click settings button
* See that there are Columns, Rounded corners, and Link to settings.
* Change setting for column
* Dismiss settings
* Reopen settings and see that settings are saved
* Repeat for Rounded corners
* Repeat for Link to
